test/core/modules/histogram.js

```diff
@@ -1,4 +1,4 @@
 const testModule = require('../templates/module-test'),
   benchmark = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQAAAAEACAYAAABccqhmAAAAAklEQVR4AewaftIAAAQeSURBVO3BQRECARDEwGT9K8ATmg5E8KEm3T5fJJl0JJl1JJl1JJl1JJl1JJl1JJl1JJl1JJl1JJl15OfevEj+wZFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFk1pFklsCjoqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqJyJJl1JJl1JJl1JJl1JJl1JJl1JJl1JJl1JJn1ATXuC15XfVDtAAAAAElFTkSuQmCC';
 
-testModule('histogram', {}, benchmark);
\ No newline at end of file
+// testModule('histogram', {}, benchmark);
```

Comment:
Why?

Reply:
Histogram module is not working, as of now and its test still passes. This was the reason why I remove looks-same module because for some reason its not functioning properly.

Reply:
Oh.